src/WCSimConstructPMT.cc

```diff
@@ -64,6 +64,9 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructPMT(G4String PMTName, G4Str
   //Optional reflectorCone:
   G4double reflectorRadius = radius + id_reflector_height * tan(id_reflector_angle); // PMT radius+ r = h * tan (theta)
   G4double reflectorThickness = 0.344*CLHEP::mm; // the actual reflector thickness is 0.5 mm but due to solid works design, I am taking the horizontal component to calculate the reflector radius. Refer the CAD drawing.
+#ifdef MERGE
```

Comment:
If you agree, I even propose to remove it. I thought it would be useful for your checks, but think it can be removed and people can use old version if needed. Let me remove it.

Reply:
Done in the commit below